engine/src/main/java/io/github/trimax/venta/engine/enums/FogType.java

```diff
@@ -0,0 +1,13 @@
+package io.github.trimax.venta.engine.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor(access = lombok.AccessLevel.PRIVATE)
+public enum FogType {
```

Comment:
I believe since we decided to support only the exponential fog model, we can freely delete this class.